packages/quantic/cypress/e2e/default-2/generatedAnswer/generated-answer.cypress.ts

```diff
@@ -772,107 +832,25 @@ describe('quantic-generated-answer', () => {
+                        unknownOption,
+                        feedbackQuestionsIndexes.correctTopic
                       );
-                      performSearch();
-                      Actions.dislikeGeneratedAnswer();
-                      Expect.displayFeedbackModal(false);
-                    }
-                  );
-
-                  scope(
-                    'when trying to open the feedback modal after rephrasing the generated answer',
-                    () => {
-                      const secondStreamId = crypto.randomUUID();
-                      const secondResponseId = crypto.randomUUID();
-
-                      mockSearchWithGeneratedAnswer(
-                        secondStreamId,
-                        param.useCase,
-                        secondResponseId
+                      Actions.clickFeedbackOption(
+                        yesOption,
+                        feedbackQuestionsIndexes.hallucinationFree
                       );
-                      mockStreamResponse(
-                        secondStreamId,
-                        genQaMessageTypePayload
+                      Actions.clickFeedbackOption(
+                        yesOption,
+                        feedbackQuestionsIndexes.documented
                       );
-                      Actions.clickRephraseButton(rephraseOptions[0]);
-                      Actions.dislikeGeneratedAnswer();
-                      Expect.displayFeedbackModal(true);
                       Actions.clickFeedbackOption(
-                        feedbackOptions.indexOf(otherOption)
+                        noOption,
+                        feedbackQuestionsIndexes.readable
                       );
-                      Actions.typeInFeedbackDetailsInput(exampleDetails);
-                      Actions.clickFeedbackSubmitButton();
-                      if (analyticsMode === 'next') {
-                        NextAnalyticsExpectations.emitQnaSubmitFeedbackReasonEvent(
-                          {
-                            feedback: {
-                              liked: false,
-                              details: exampleDetails,
-                              reason: 'other',
-                            },
-                            answer: {
-                              responseId: secondResponseId,
-                              type: answerType,
-                            },
-                          },
-                          exampleTrackingId
-                        );
-                      } else {
-                        Expect.logGeneratedAnswerFeedbackSubmit(
-                          secondStreamId,
-                          {
-                            reason: otherOption,
-                            details: exampleDetails,
-                          }
-                        );
-                      }
 
+                      Actions.clickFeedbackSubmitButton();
                       Actions.clickFeedbackDoneButton();
```

Comment:
are you not expecting something here? why do all these actions to expect nothing?

Reply:
No cause mainly these actions are to prepare for the next step of the test and the expectations that need to be checked after submitting a feedback are already check in another test so no need to re add them here.